regexes/client/browsers.yml

```diff
@@ -5,6 +5,76 @@
 # @license http://www.gnu.org/licenses/lgpl.html LGPL v3 or later
 ###############
 
+# Sushi Browser (https://sushi-browser.com/)
+- regex: 'SushiBrowser/(\d+[\.\d]+)'
+  name: 'Sushi Browser'
+  version: '$1'
+  engine:
+    default: 'Blink'
+
+# Peeps dBrowser (https://peepsx.com/dbrowser.html)
+- regex: 'dBrowser/(\d+[\.\d]+)'
+  name: 'Peeps dBrowser'
+  version: '$1'
+  engine:
+    default: 'Blink'
+
+# LT Browser (https://www.lambdatest.com/lt-browser)
+- regex: 'LTBrowser/(\d+[\.\d]+)'
+  name: 'LT Browser'
+  version: '$1'
+  engine:
+    default: 'Blink'
+
+# Landis+Gyr AIM Browser (https://www.landisgyr.eu/product/gridstream-aim/)
+- regex: 'LandisGyrAIMbrowser/(\d+[\.\d]+)'
+  name: 'Landis+Gyr AIM Browser'
+  version: '$1'
+  engine:
+    default: 'Blink'
```

Comment:
there is no address bar in this application to go to other sites.

client type: `app`